app/components/navigation/navbar_component.html.erb

```diff
@@ -5,7 +5,7 @@
             <ul>
                 <%= nav_link("Home", '/') %>
                 <% resources.each do |resource| %>
-                    <%= nav_link(resource[:front_matter][:title], resource[:path]) %>
+                    <%= nav_link(resource.title, resource.path) %>
                 <% end %>
                 <%= nav_link("Teaching events", events_path) %>
```

Comment:
Are these two additional `nav_link` references going to be artefacts that need removing?

Reply:
Yeah eventually, this PR shouldn't have any visual impact at all though.